Kiosk/Images.xcassets/StubbingFlag.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "StagingFlag@2x.png",
```

Comment:
I think this image should probably look different, give the S for staging - I took a quick stab at this

![stubs](https://cloud.githubusercontent.com/assets/49038/10446502/552dc79c-7175-11e5-89b4-2e8da666da81.png)